File: src/spring/example/src/main/java/com/murilobeltrame/example/WeatherForecast.java
package com.murilobeltrame.example;

import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

public class WeatherForecast {
    @Getter
    @Setter
    private Instant date;

    @Getter
    @Setter
    private int temperatureC;

    @Getter
    private final int temperatureF = 32 + (int)(temperatureC / 0.5556) ;

    @Getter
    @Setter
    private String summary;

    WeatherForecast(Instant date, int temperatureC, String summary){
        setDate(date);
        setTemperatureC(temperatureC);
        setSummary(summary);
    }
}

File: src/fastapi/models/weatherforecast.py
import datetime
import string


class WeatherForecast:
    def __init__(self, date: datetime, temperature_c: int, summary: string):
        self.date = date
        self.temperature_c = temperature_c
        self.summary = summary
        self.temperature_f = int(32 + temperature_c / 0.5556)
